charts/konk/templates/kind-pod.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "konk.fullname" . }}-kind-pod
+  labels:
+    {{- include "konk.labels" . | nindent 4 }}
+spec:
+  serviceAccountName: {{ include "konk.serviceAccountName" . }}
+  containers:
+  - name: kind
+    securityContext:
+      {{- toYaml .Values.kind.securityContext | nindent 6 }}
+    image: "{{ .Values.kind.image.repository }}:{{ .Values.kind.image.tag }}"
```

Comment:
I would default this to the appVersion like your comment describes.

Reply:
Updated to default to the appVersion.